ckanext/search/tests/test_filters.py

```diff
@@ -39,65 +54,66 @@ def test_filters_unknown_top_operators():
 def test_filters_dollar_fields_escaped():
 
     filters = {"$$some_field": "some_value"}
+    search_schema = {"fields": [{"name": "$some_field"}]}
 
-    result = query_filters_validator(filters)
+    result = query_filters_validator(filters, search_schema)
     assert result and result.field == "$some_field" and result.op == "eq"
```

Comment:
If you prefer this style should also work and might give a clearer traceback on failure
```suggestion
    assert result == FilterOp("$some_field", "eq", "some_value")
```

Reply:
oh, I see you're already doing this below, my bad